src/Entity/FieldableEdgeEntityInterface.php

```diff
@@ -26,7 +26,7 @@
 /**
  * Interface for fieldable Edge entities.
  */
-interface FieldableEdgeEntityInterface extends FieldableEntityInterface, EdgeEntityInterface {
+interface FieldableEdgeEntityInterface extends \IteratorAggregate, FieldableEntityInterface, EdgeEntityInterface {
```

Comment:
Where exactly leverage the newly extended interface in your PR? I see FieldItemInterface type hint in the array_map() below.

Reply:
If a fieldable entity does not implement `Traversable`, then the form display's `extractFormValues()` will always return an empty array (and a warning), since it tries to traverse the entity's fields with a foreach.